src/scripts/main-nav.js

```diff
@@ -117,18 +117,27 @@
     }
   }
 
-  function getLoginElem (nickname, avatarImageURL) {
+  function getLoginElem (id, nickname, githubAvatar) {
+    if (nickname) {
+      // github user so show their nickname and avatar
+      var avatarImageURL = githubAvatar;
+      var label = nickname;
+    } else {
+      // email/password user so just show 'account' and default avatar
+      var avatarImageURL = 'common/styleguide/images/default-avatar.png';
+      var label = 'Account';
+    }
```

Comment:
We don't have access to name, and I discussed with ekta whether it was viable to put the email address or some shortened form of it in the navbar and she preferred to go with 'Account' at least initially.

That simplification is actually the way I initially wrote it and then I thought maybe the if/else was clearer. I'll switch it back.